### PR TITLE
Testing with Travis 2

### PR DESCRIPTION
**Pull request by BrandonMathis**
## What Changed
* Added tests for Rails Ansible playbook

## Why
* I kept breaking tape so I've written a Dockerfile that sets up a container and runs tape inside it against a vanilla rails app

### CODE REVIEW

File: test/rails/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
+FROM brandonmathis/taperole:latest
```
Comment (dkniffin) [inline test/rails/Dockerfile]: I'm a complete docker noob, but does this mean the version of tape that is being tested is your fork? I think we want to do `smashingboxes/taperole`, and somehow, pull in the PR's version of it.

**Comment by dkniffin** [comment]
It looks like a lot of the files in this PR are a copy of vanilla-rails-app. Is that intentional? Do we want to include vanilla-rails-app in tape's source?

File: test/rails/Dockerfile (hunk 1 above)
Comment (BrandonMathis) [inline test/rails/Dockerfile]: Actually. This is a docker base box that I defined which is a base install of ubuntu with ruby 2.3.0 and ansible.

[It's hosted here on docker hub](https://hub.docker.com/r/brandonmathis/taperole/)

**Comment by BrandonMathis** [comment]
@dkniffin I think I do for testing purposes. I've been using that vanilla rails app on github tho so I guess I could just clone that down instead of including it in source. What do you think?

**Comment by dkniffin** [comment]
I think it should be cloned down for the tests, because it's not really part of tape. Also, if you include it, it'll get downloaded every time someone does `gem install taperole`

**Pull request merged**